deployment/config/public-graphql-api-gateway/gateway.config.ts

```diff
@@ -0,0 +1,15 @@
+// @ts-expect-error not a dependency
```

Comment:
I think you can add as a pinned devDependency? The way renovate will update it, and you don't need to have it statically defined in this line: https://github.com/graphql-hive/console/blob/experiment-federation/deployment/services/public-graphql-api-gateway.ts#L15 (you can read `deployment/package.json` in Pulumi code and take the version from it)

Reply:
no please not there will be a dependency cycle issue since gateway depends on packages in this repo.